pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseRealtimeClusterIntegrationTest.java

```diff
@@ -92,9 +92,11 @@ protected void overrideServerConf(PinotConfiguration configuration) {
   protected List<String> getNoDictionaryColumns() {
     // Randomly set time column as no dictionary column.
     if (new Random().nextInt(2) == 0) {
-      return Arrays.asList("ActualElapsedTime", "ArrDelay", "DepDelay", "CRSDepTime", "DaysSinceEpoch");
+      return Arrays.asList("ActualElapsedTime", "ArrDelay", "DepDelay", "CRSDepTime", "RandomAirports",
+          "DivTotalGTimes", "DaysSinceEpoch");
     } else {
-      return super.getNoDictionaryColumns();
+      return Arrays.asList("ActualElapsedTime", "ArrDelay", "DepDelay", "CRSDepTime", "RandomAirports",
```

Comment:
Suggest modifying the one in the `BaseClusterIntegrationTest` so that it is covered for offline tests as well

Reply:
I tried adding it earlier. There are some tests extending BaseClusterIntegrationTest that fail because they work only on SV columns. I'll take this up as a separate change if that's ok?